<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(36, 59, 85);</string>
  </property>
  <widget class="QPushButton" name="button_validation">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);
font: 75 8pt &quot;System&quot;;
color: rgb(255, 255, 255);
background-color: rgb(133, 147, 152);
</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Image_Validation">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;Calisto MT&quot;;</string>
   </property>
   <property name="text">
    <string>     Add Image for validation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_image_data">
   <property name="geometry">
    <rect>
     <x>824</x>
     <y>50</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 10pt &quot;Calisto MT&quot;;</string>
   </property>
   <property name="text">
    <string>        Image from database</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_data">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 127);
font: 75 8pt &quot;System&quot;;
color: rgb(255, 255, 255);
background-color: rgb(133, 147, 152);
</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_validation">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>241</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_data">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>150</y>
     <width>241</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="person_recognized_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(9, 234, 24);
font: 75 22pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Person Identified</string>
   </property>
  </widget>
  <widget class="QLabel" name="person_name_label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="person_not_recognized_label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;System&quot;;
color: rgb(220, 36, 36);
color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>Person Not Identified</string>
   </property>
  </widget>
  <widget class="QLabel" name="x_label">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 26pt &quot;MS Shell Dlg 2&quot;;
color: rgb(220, 36, 36);
color: rgb(255, 0, 0);
</string>
   </property>
   <property name="text">
    <string>     X</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Search for person in Database</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="compareButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Compare two images if they are for same person</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Image needs to be decrypted?</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
